recipes/glibmm/all/conanfile.py

```diff
@@ -189,11 +184,11 @@ def package_info(self):
         if self._abi_version == "2.68":
             self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::sigc++"]
         else:
-            self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::sigc++-2.0"]
+            self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::libsigcpp"]
 
         self.cpp_info.components[giomm_component].set_property("pkg_config_name", giomm_component)
         self.cpp_info.components[giomm_component].libs = [giomm_component]
-        self.cpp_info.components[giomm_component].includedirs = [ os.path.join("include", giomm_component)]
+        self.cpp_info.components[giomm_component].includedirs = [os.path.join("include", giomm_component)]
         self.cpp_info.components[giomm_component].requires = [glibmm_component, "glib::gio-2.0"]
 
     def package_id(self):
```

Comment:
I can't review the line below, but there is a `if not self.options["glib"].shared` here to replace by `if not self.dependencies["glib"].options.shared`